src/neuroconv/nwbconverter.py

```diff
@@ -111,9 +111,16 @@ def _validate_source_data(self, source_data: Dict[str, dict], verbose: bool = Tr
     def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optional[dict] = None) -> None:
         conversion_options = conversion_options or dict()
         for interface_name, data_interface in self.data_interface_objects.items():
-            data_interface.add_to_nwbfile(
-                nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
-            )
+            if isinstance(data_interface, NWBConverter):
+                data_interface.add_to_nwbfile(
+                    nwbfile=nwbfile,
+                    metadata=metadata,
+                    conversion_options=conversion_options.get(interface_name, dict()),
+                )
+            else:
+                data_interface.add_to_nwbfile(
+                    nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
+                )
```

Comment:
@CodyCBakerPhD 
the `conversion_options` for `MiniscopeConverter` would look like this when using it from inside an `NWBConverter`:
`conversion_options = dict(MiniscopeConverter=dict(Miniscope=dict(stub_test=True)))`
However
`**conversion_options.get(interface_name, dict())`
returns an error :

```
metadata = DeepDict: {'NWBFile': {'session_description': 'Auto-generated by neuroconv', 'identifier': '5f1bc608-b43b-4b22-a45c-20...2'}], 'ImageSeries': [{'name': 'BehavCamImageSeries', 'device': 'BehavCam2', 'dimension': [1280, 720], 'unit': 'px'}]}}
conversion_options = {'TestMiniscopeConverter': {'Miniscope': {'stub_frames': 2, 'stub_test': True}}}

    def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optional[dict] = None) -> None:
        conversion_options = conversion_options or dict()
        for interface_name, data_interface in self.data_interface_objects.items():
>           data_interface.add_to_nwbfile(
                    nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
            )
E           TypeError: add_to_nwbfile() got an unexpected keyword argument 'Miniscope'
../../../src/neuroconv/nwbconverter.py:114: TypeError
```
I'm not sure this is the best fix possible, what do you think?

Reply:
What conversion options does the `MiniscopeConverter` need to expose? Is it only `stub`?

If so the best fix will not be to modify the base NWBConverter, but do what we did with the source schema and go for a Converter class that behaves more like an Interface - define the conversion option schema explicitly as we would normally operate it on an interface, e.g., `get_schema_from_method_signature` applied to `add_to_nwbfile`

Reply:
Yes, the `MiniscopeBehaviorInterface` doesn't have any conversion options at the moment, so it's only what `MiniscopeImagingInterface` exposes in `add_to_nwbfile()` : `photon_series_type`, `stub_test`, `stub_frames`
See in https://github.com/catalystneuro/neuroconv/pull/498/commits/af2818b1e1559ec5f023be1680eb5e8f096d17af

Reply:
> photon_series_type, stub_test, stub_frames

For Minscope, shouldn't we be pinning `photon_series_type` to `OnePhotonSeries`?